apps/05Mastermind.js

```diff
@@ -42,8 +42,8 @@ function generateHint(solution, guess) {
       }
     }
 
-    for (index=0; index < guessArray.length; index++) {
```

Comment:
And... Again, the red lines 45 & 46 are also not in my pre-commit code. So! What am I doing wrong?? I don't like seeing red X's by my submissions...